docs/nconnect-get-started-cli.md

```diff
@@ -0,0 +1,154 @@
+# Getting Started
+
+nConnect allows you to securely connect to remote machines without the need for a server, public IP address, or publicly exposed ports. It features end-to-end encryption for the highest level of security and multi-path aggregation for maximum throughput.
+
+Let's connect two devices using nConnect. We'll use `ping` and `ssh` to test the connection. Any two devices that can run nConnect will do: cloud server, home desktop, embedded Linux device, NAS device, mobile phone, etc.
+
+Both devices can be in different locations, home or office, cafe shop or cloud server base.
+
+The rough outline is:
+- Set up nConnect Server
+- Set up nConnect Client
+- SSH** Server from Client
```

Comment:
What's this **?

Reply:
Sorry, nconnect-get-started-cli.md is removed. because its content are moved into nconnect-get-started.md and nconnect-build-from-source-code.md